### DISCUSSION TITLE
[QUESTION]: Change color dynamicaly on each day in the calendar

### DISCUSSION THREAD

**Discussion by andreiwpopa** [coolsam726/flatpickr#73] [Q&A]
### What happened?

Basically the title. I tried initializing flatpickr in the blade file, and add a class name for each day, but is not working, because it overwritten, and the class doesn't remain there. 

Any suggestions on how i can do it ? 

### How to reproduce the bug

Basically the title.

### Package Version

2.0.0

### PHP Version

8.3

### Laravel Version

10

**Comment by jimiero** [comment]
@coolsam726 any plans for this? I'd like to be able to highlight the holidays dates with a color
